test/unittests/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ find_path(GMOCK_INCLUDE_DIR gmock/gmock.h
           PATHS /usr/includes
           PATH_SUFFIXES gmock)
 
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-conversion-null")
+
```

Comment:
Should be added only in the "branch" in which this is needed (I understand that an specific Ubuntu version), in the `IF ()` part.